suite/smoke/vrf_test.go

```diff
@@ -36,21 +41,17 @@ var _ = Describe("VRF suite @vrf", func() {
 
 	BeforeEach(func() {
 		By("Deploying the environment", func() {
-			e, err = environment.DeployOrLoadEnvironment(
-				environment.NewChainlinkConfig(
-					config.ChainlinkVals(),
-					"chainlink-vrf",
-					config.GethNetworks()...,
-				),
-			)
-			Expect(err).ShouldNot(HaveOccurred(), "Environment deployment shouldn't fail")
-			err = e.ConnectAll()
-			Expect(err).ShouldNot(HaveOccurred(), "Connecting to all nodes shouldn't fail")
+			e = environment.New(nil).
```

Comment:
Is there any way to provide namespace names?

Are simulated geth instances always launched by default now?

Reply:
Sure, here is the config for now
// Config environment configuration
type Config struct {
	TTL               time.Duration
	NamespacePrefix   string
	Namespace         string
	Labels            []string
	NSLabels          *map[string]*string
	ReadyCheckData    *client.ReadyCheckData
	DryRun            bool
	InsideK8s         bool
	KeepConnection    bool
	RemoveOnInterrupt bool
}

Reply:
Yes, we should just use table tests to inject anything you want and then run that particular case, like:
<img width="838" alt="image" src="https://user-images.githubusercontent.com/14962379/174970326-d5e10090-51f3-42e4-a510-13dc979c73ca.png">

Reply:
We can also tear down `Networks` and `networks.yaml` because in fact, when we need different clients + deployments ( in external network case I still would like to have no-op deployment just to contain URLs and params too ) and it's a single network case - we can use table test an just inject both client with config and deployment you want, making that's explicit just adds to readability

If CL requires multiple networks, that'll be a completely different suite.

Reply:
Because when we have one client - we don't need `Networks` wrapper, when we have 1+ we don't need `DefaultNetwork`, just make client calls explicitly, make things simpler